bin/fin

```diff
@@ -2650,15 +2650,10 @@ configure_resolver_linux () {
 
 	# TODO: split resolver and network configuration into different functions
 	# Adding a subnet for Docksal. Make sure we don't do this twice
-	if ! grep -q "$DOCKSAL_DEFAULT_SUBNET" /etc/network/interfaces; then
+  if ! grep -q 'docksal' `nmcli dev status`; then
 		echo-green "Adding a subnet for Docksal..."
-		cat > /tmp/docksal.ip.addr <<EOF
-	up   ip addr add ${DOCKSAL_DEFAULT_SUBNET} dev lo label lo:docksal
-	down ip addr del ${DOCKSAL_DEFAULT_SUBNET} dev lo label lo:docksal
-	dns-nameservers ${DOCKSAL_DEFAULT_IP}
-EOF
-		sudo sed -i '/iface lo inet loopback/r /tmp/docksal.ip.addr' /etc/network/interfaces
-		rm -f /tmp/docksal.ip.addr
+    nmcli con add con-name lo:docksal ifname docksal0 type bridge ip4 ${DOCKSAL_DEFAULT_SUBNET}
```

Comment:
I don't think this works as anticipated:

```
root@ubuntu-512mb-sfo2-01:~# nmcli con add con-name lo:docksal ifname docksal0 type bridge ip4 192.168.64.1/24
Connection 'lo:docksal' (79ff1fa9-674d-4ee8-8072-185022c9cc0d) successfully added.
root@ubuntu-512mb-sfo2-01:~# nmcli con modify lo:docksal ipv4.dns 192.168.64.100
```

I see the settings are added in `/etc/network/interfaces`:

```
root@ubuntu-512mb-sfo2-01:~# cat /etc/network/interfaces 
# This file describes the network interfaces available on your system
# and how to activate them. For more information, see interfaces(5).

# The loopback network interface
auto lo
iface lo inet loopback
	up   ip addr add 192.168.64.1/24 dev lo label lo:docksal
	down ip addr del 192.168.64.1/24 dev lo label lo:docksal
	dns-nameservers 192.168.64.100

source /etc/network/interfaces.d/*
```

Unfortunately these do not get applied right away, neither after a reboot, so neither of these IPs can be reached:

```
192.168.64.1
192.168.64.100
```

Reply:
Scratch the part about settings in `/etc/network/interfaces`. Those were added after I ran Docksal installation...